curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md

```diff
@@ -109,11 +109,11 @@ What is the difference between `let` and `const` variable declarations in JavaSc
 
 #### --distractors--
 
-`let` is block scoped, `const` is not.
+
 
 ---
 
-`const` is block scoped, `let` is not.
+
```

Comment:
How about a distractor related to `hoisting`?

Reply:
I don't remember hoisting mentioned in the lectures